scripts/eval/run_and_process_phenix_clashscore.py

```
@@ -0,0 +1,136 @@
+
+
+def process_one_experiment(experiment: Experiment) -> pd.DataFrame:
+    # make sure there are no nan lines in the CIF file; this is an extra
+    # precaution, even though our CIF writers should now avoid writing nans
+    file_with_no_nans = experiment.refined_cif_path.parent / "nonan.cif"
+    json_output = experiment.refined_cif_path.parent / "clashscore.json"
+    logfile = experiment.refined_cif_path.parent / "clashscore.log"
+    logger.info(f"Removing nans from {experiment.refined_cif_path}")
+
+    with file_with_no_nans.open("w") as fn:
+        retcode = subprocess.call(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)